themes/bootstrap3/js/record.js

```diff
@@ -289,6 +289,24 @@ function applyRecordTabHash() {
 
 $(window).on('hashchange', applyRecordTabHash);
 
+function moreLessSubjects() {
+  $('.more-subjects, .less-subjects').off('click');
+  $('.more-subjects').click(function moreSubjects() {
+    $('.subject-hidden').removeClass('hidden');
+    $('#less-subjects').removeClass('hidden');
+    $('#more-subjects').addClass('hidden');
+    return false;
+  });
+
+  $('.less-subjects').click(function lessFacets() {
```

Comment:
There could be a more appropriate name for the function. :)

Reply:
I have received occasional complaints from people who think the more/less controls should be more/fewer controls. Depends on the grammatical context you mentally place around the word. ;-) But I agree that maybe function names containing show or hide would be more informative.